macros/compare_all_columns.sql

```diff
@@ -0,0 +1,59 @@
+{% macro compare_all_columns(model_name, primary_key, prod_schema, exclude_columns, updated_at_column, exclude_recent_hours, direct_conflicts_only) -%}
+  {{ return(adapter.dispatch('compare_all_columns', 'audit_helper')( model_name, primary_key, prod_schema, exclude_columns, updated_at_column, exclude_recent_hours, direct_conflicts_only )) }}
+{%- endmacro %}
+
+{% macro default__compare_all_columns( model_name, primary_key, prod_schema, exclude_columns, updated_at_column, exclude_recent_hours, direct_conflicts_only ) -%}
+
+  {% set columns_to_compare=audit_helper.pop_columns(model_name, exclude_columns) %}
+
+  {% set old_etl_relation_query %}
+      select * from {{prod_schema}}.{{ model_name }}
```

Comment:
Reading [api.Relation.create docs](https://docs.getdbt.com/reference/dbt-classes#relation) and found this:
> This object should always be used instead of interpolating values with {{ schema }}.{{ table }} directly.

so ... "I'm in these docs and I don't like it"

Reply:
OK, I'm not totally sure if the way I've updated the code this fully addresses the issue you raise, but please have a look, I think it may do the trick.